packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -417,6 +418,36 @@ pool.addClient('app-client', {
 });
 ```
 
+### Identity Providers
+
+Users that are part of a user pool can sign in either directly through a user pool, or federate through a third-party
+identity provider. Once configured, the Cognito backend will take care of integrating with the third-party provider.
+Read more about [Adding User Pool Sign-in Through a Third
+Party](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-identity-federation.html).
+
+The following third-party identity providers are currentlhy supported in the CDK -
```

Comment:
```suggestion
The following third-party identity providers are currently supported in the CDK -
```

Reply:
Shouldn't we say something like "there are convenience methods for ... you can implement your own by ...".

Reply:
(Why do we not have Google? Is it hard to add?)

Reply:
Also maybe a generic `oidc` provider?

Reply:
I've cut those those out with the intent to keep the PR small and manageable.

I'll likely add the generic OIDC in a separate PR and leave the rest to the community.